src/pages/Introduce/introduce.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./introduce.css";
+import { Link } from "react-router-dom";
+
+const handleInput = () => {
+  console.log("input 태그 동작");
+};
+const handleButton = () => {
+  console.log("button 태그 동작");
+};
+
+const Introduce = () => {
+  return (
+    <>
+      <div id="wrap">
+        <div id="Black">
```

Comment:
```suggestion
        <div className="black">
```
첫글자가 대문자로 시작하는 것은 컴포넌트의 이름을 지을 때 사용되는 컨벤션입니다!
className은 다른 컨벤션으로 적용해주시고
className만 봐도 어떤 역할을 하는 요소인지 알 수 있도록 의미있게 이름을 지어주세요!